packages/eslint-plugin-next/lib/index.js

```diff
@@ -39,12 +40,13 @@ module.exports = {
         '@next/next/next-script-for-ga': 1,
         '@next/next/no-document-import-in-page': 2,
         '@next/next/no-head-import-in-document': 2,
-        '@next/next/no-script-in-document': 2,
+        '@next/next/no-script-in-document': 0,
```

Comment:
If we're turning off this rule entirely, we should probably just delete it completely (unless you think there's a reason why folks would like to turn it on?)